src/services/pasteEdits.ts

```diff
@@ -1,4 +1,6 @@
-import { findIndex } from "../compiler/core.js";

```

Comment:
Woah, is this line in main? We should fix this, this kind of import is most likely broken...

Reply:
There are others wrong than just this one, working on a PR quick...

Reply:
https://github.com/microsoft/TypeScript/pull/59544